### PR TITLE
Compile groovy scripts

### PR DESCRIPTION
**Pull request by g4s8**
#747 - moved stakeholder groovy scripts to `src/main/groovy`, compile them with GMaven, added `StkRuntime` to run stakeholder from `Script` instance.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#949](https://github.com/zerocracy/farm/issues/949) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#949](https://github.com/zerocracy/farm/issues/949) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1091193165); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer

File: src/main/groovy/com/zerocracy/package-info.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ */
+
+/**
+ * Groovy scripts.
+ *
+ * @author Kirill (g4s8.public@gmail.com)
+ * @version $Id$
+ * @since 0.22
+ */
+package com.zerocracy;
```
Comment (carlosmiranda) [inline src/main/groovy/com/zerocracy/package-info.java]: @g4s8 shouldn't this be in `com.zerocracy.stk`?

File: src/test/java/com/zerocracy/farm/reactive/BrigadeTest.java [hunk 2]
```diff
@@ -88,15 +87,14 @@ public void parsesGroovy() throws Exception {
     }
 
     @Test
-    public void parsesGroovyScript() throws Exception {
+    public void runGroovyScript() throws Exception {
```
Comment (carlosmiranda) [inline src/test/java/com/zerocracy/farm/reactive/BrigadeTest.java]: @g4s8 subtle change... let's call this `runsGroovyScript`

File: pom.xml [hunk 3]
```diff
@@ -610,7 +610,12 @@ SOFTWARE.
                         <limit>
                           <counter>INSTRUCTION</counter>
                           <value>COVEREDRATIO</value>
-                          <minimum>0.60</minimum>
+                          <minimum>0.56</minimum>
+                          <!--
+                            @todo #747:30min Groovy scripts are included in instruction coverage
+                             metric, lets add more tests for them and change minimum value
+                             back to 0.60
```
Comment (carlosmiranda) [inline pom.xml]: @g4s8 instead of lowering the coverage, let's temporarily exclude `com.zerocracy.stk` and have performer of puzzle fix the issue. This way we don't get other classes accidentally regressing in coverage.

**Comment by g4s8** [comment]
@carlosmiranda fixed in 9571d36

File: pom.xml [hunk 4]
```diff
@@ -648,6 +648,12 @@ SOFTWARE.
                   <excludes combine.children="append">
                     <exclude>com/zerocracy/pmo/People.java</exclude>
                     <exclude>com/zerocracy/pings/*.*</exclude>
+                    <!--
+                      @todo #747:30min Groovy scripts are included in instruction coverage
```
Comment (carlosmiranda) [inline pom.xml]: @g4s8 puzzle description is wrong now:

* Groovy scripts are _excluded_ from the coverage
* The minimun value is already 0.60
* What we need to do is remove the exclusion while retaining the coverage at 0.60 (or above) somehow

Makes sense?

**Comment by carlosmiranda** [comment]
@g4s8 just need to update puzzle, see above

**Comment by g4s8** [comment]
@carlosmiranda right, fixed 98cb7b1

**Comment by carlosmiranda** [comment]
@g4s8 looks good, but there are merge conflicts now

**Comment by g4s8** [comment]
@carlosmiranda merged `master` and resolved conflicts

**Comment by carlosmiranda** [comment]
@g4s8 there are some failed tests in Travis, can you check?

**Comment by 0crat** [comment]
@carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by carlosmiranda** [comment]
@g4s8 ping

**Comment by g4s8** [comment]
@carlosmiranda fixed with merge 9888b44

**Comment by g4s8** [comment]
@yegor256 could you please look at this PR? Each new push to master with groovy script changes causes new conflicts.

**Comment by yegor256** [comment]
@rultor merge

**Comment by g4s8** [comment]
@yegor256 should be fixed now, please restart merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by g4s8** [comment]
@yegor256 resolved conflicts, restart Rultor please

**Comment by yegor256** [comment]
@g4s8 sorry, conflicts again. I won't push anything to master for a while, please fix and let's merge

**Comment by carlosmiranda** [comment]
@g4s8 conflicts again :(

**Comment by g4s8** [comment]
@carlosmiranda I'm waiting for @yegor256 response to continue to resolve conflicts: https://github.com/zerocracy/farm/issues/747#issuecomment-387305263

**Comment by yegor256** [comment]
@g4s8 boosted

**Comment by g4s8** [comment]
@yegor256 merged master, resolved conflicts, can you restart merge?

**Comment by g4s8** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#949](https://github.com/zerocracy/farm/issues/949) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@0crat quality good
